prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+var languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+var firstElement = languages[0];
+console.log('The first element of the array is ' + firstElement);
+
+var thirdElement = languages[2];
+console.log('The first element of the array is ' + thirdElement);
+
+var length = languages.length;
+console.log('The length of the array is ' + length);
+
+var lastElement = languages[5];
+console.log('The last element of the array is ' + lastElement);
```

Comment:
You can get the last element of any array by subtracting `1` from its length without having to know for sure how long it it.

```js
var lastIndex = languages.length - 1;
```

Then you can use that variable to get the last element

```js
var lastLanguage = languages[lastIndex];
```

Reply:
![image](https://user-images.githubusercontent.com/99709530/156656923-9e67858e-317c-410c-a71c-5c55683d87e1.png)